docker-compose.yml

```diff
@@ -11,8 +11,8 @@ services:
     build:
       context: .
       cache_from:
-        - batdevcontainerregistry.azurecr.io/manage-courses-backend:latest
-    image: batdevcontainerregistry.azurecr.io/manage-courses-backend:latest
+        - dfedigital/manage-courses-backend:latest
+    image: dfedigital/manage-courses-backend:latest
```

Comment:
This seems unrelated to the other changes, changing the container registry for the mcbe image.

Reply:
I just didn't like the ider of them being on separate registries. May just revert for now.